swift/apple/FirezoneKit/Sources/FirezoneKit/Views/MenuBar.swift

```diff
@@ -1,433 +1,394 @@
 //
 //  MenuBar.swift
-//  (c) 2024 Firezone, Inc.
-//  LICENSE: Apache-2.0
+//
+//
+//  Created by Jamil Bou Kheir on 4/2/24.
 //
 
-#if os(macOS)
-  import Combine
-  import Dependencies
-  import OSLog
-  import SwiftUI
-  import NetworkExtension
-
-  @MainActor
-  public final class MenuBar: NSObject {
-    @Dependency(\.mainQueue) private var mainQueue
-
-    private var cancellables: Set<AnyCancellable> = []
-    private var statusItem: NSStatusItem
-
-    private var resources: [Resource]?
-    private var isMenuVisible = false {
-      didSet { handleMenuVisibilityOrStatusChanged(status: tunnelStore.status) }
-    }
-    private lazy var signedOutIcon = NSImage(named: "MenuBarIconSignedOut")
-    private lazy var signedInConnectedIcon = NSImage(named: "MenuBarIconSignedInConnected")
-
-    private lazy var connectingAnimationImages = [
-      NSImage(named: "MenuBarIconConnecting1"),
-      NSImage(named: "MenuBarIconConnecting2"),
-      NSImage(named: "MenuBarIconConnecting3"),
-    ]
-    private var connectingAnimationImageIndex: Int = 0
-    private var connectingAnimationTimer: Timer?
-
-    private var tunnelStore: TunnelStore
-    private var settingsViewModel: SettingsViewModel
-    private var logger: AppLogger
-
-    public init(tunnelStore: TunnelStore, settingsViewModel: SettingsViewModel, logger: AppLogger) {
-      self.statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.variableLength)
-      self.tunnelStore = tunnelStore
-      self.settingsViewModel = settingsViewModel
-      self.logger = logger
-      super.init()
-
-      if let button = statusItem.button {
-        button.image = signedOutIcon
-      }
-
-      createMenu()
-      setupObservers()
-    }
```

Comment:
All of the stuff in here that looks like it belongs in a model was moved to `SessionViewModel` which is shared with iOS.

iOS uses `SessionView` and macOS uses `MenuBar` -- both present the same data.